### ISSUE TITLE
TODO: Remaining Hunter Pet Trainer Issues after #815

### ISSUE THREAD

**Issue by GrenderG** [The-Alpha-Project/alpha-core#834]
This is the remaining issues (and new ones) from https://github.com/The-Alpha-Project/alpha-core/issues/789 after applying https://github.com/The-Alpha-Project/alpha-core/pull/815.

1. Some of the trainers are using weapons, we are still missing them after applying the PR (please check all images of https://github.com/The-Alpha-Project/alpha-core/issues/789 to confirm which ones are missing weapons).
2. Some trainers have wrong level / faction / stats (hp, dmg...). We need to double check all of them in order to fix them. For example, Rebald Yorglun is currently level 1 with no faction (this is just an example, there are more issues / inconsistencies).
3. @dirtbikercj wasn't able to find a suitable spawn for Nerra, because the reference mob Radnaal Maneweaver entry: 6287 is spawned underground and not in the spot he is suppose to be.

**Comment by dirtbikercj** [comment]
Ill add this here aswell, seems more appropriate than opening a new issue. 

<img width="418" alt="image" src="https://user-images.githubusercontent.com/62989449/209989970-7853fbdb-a5c3-4236-bc8e-97594a9abae2.png">

@geo-tp will be opening a PR to correct

**Comment by mindphluxnet** [comment]
#902 fixes the faction for the trainer "Frank Ward" in Westfall

**Comment by Fluglow** [comment]
Some pet trainers should have their pet summoned by casting a summon spell. I'm guessing these should be spawned for trainers that don't have a named pet (#582). Support for this was added in #919, but the trainers are missing the summon spells in their `creature_template` entries (spell ID columns). The spells used for this are most likely IDs 7903-7913.
![image](https://user-images.githubusercontent.com/38987331/221873241-06eec3e7-2326-4315-be70-0d81951a3a66.png)
